src/renderer/qssWidget/js/qssMenuWidget.js

```diff
@@ -195,16 +200,23 @@
      * @param {Any} value - The new value of the setting in the QSS menu.
      * @param {KeyboardEvent} keyboardEvent - The keyboard event (if any) that led to the
      * change in the setting's value.
+     * @param {Boolean} closeOnSelect - Flag to know if the menu must be closed on select
      */
-    gpii.qssWidget.menu.updateValue = function (that, menu, container, value, keyboardEvent) {
+    gpii.qssWidget.menu.updateValue = function (that, menu, container, value, keyboardEvent, closeOnSelect) {
+        // we are closing the menu by default
+        var closeNow = (fluid.isValue(closeOnSelect) && closeOnSelect === false) ? false : true;
```

Comment:
This can just be var closeNow = closeOnSelect !== false - you do love over-expressing these checks : P

Reply:
True, true -- it shows that I'm coming from PHP -- we are checking everything, like 3 times, just in case 🛩